pkg/operator/target_config_reconciler.go

```diff
@@ -74,9 +74,11 @@ type TargetConfigReconciler struct {
 	queue                      workqueue.TypedRateLimitingInterface[queueItem]
 	kubeInformersForNamespaces v1helpers.KubeInformersForNamespaces
 	crdClient                  apiextv1.ApiextensionsV1Interface
+	crdInformer                apiextinformer.SharedInformerFactory
 	operatorNamespace          string
 	resourceCache              resourceapply.ResourceCache
 	kueueImage                 string
+	serviceMonitorSupport      bool
```

Comment:
I don't really understand the motivation between serviceMonitorSupport.

Is this to deploy on a non ocp cluster?